AUTHORS.txt

```diff
@@ -342,6 +345,8 @@ Jason R. Coombs
 JasonMo
 JasonMo1
 Jay Graves
+Jean Abou Samra
+Jean Abou-Samra
```

Comment:
```suggestion
```

I think the other one's correct ;-)

Reply:
This AUTHORS.txt is auto generated. @jeanas if you like, feel free to submit a PR to the `.mailmap` file.